backend/utils/pubsub.js

```diff
@@ -0,0 +1,253 @@
+/* globals logger */
```

Comment:
One issue with file path: this is the only file under a `utils` folder, and it isn't a utility. Can you move the file under `backup/` with the rest of the core server files and delete the `utils` folder since there won't be anything else under it?